terraform/modules/cloudtrail/cloudtrail/outputs.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
Can this be deleted if we're not outputting anything?

Reply:
Yeah I added this because of this bit in the docs:
https://www.terraform.io/docs/modules/create.html#standard-module-structure
>>> main.tf, variables.tf, outputs.tf. These are the recommended filenames for a minimal module, even if they're empty. main.tf should be the primary entrypoint. For a simple module, this may be where all the resources are created. For a complex module, resource creation may be split into multiple files but all nested module usage should be in the main file. variables.tf and outputs.tf should contain the declarations for variables and outputs, respectively.

But it's not consistent with the rest of our modules and it doesn't do anything so actually I'm going to get rid. If we're going to have empty files just to indicate no input and no output then we should do it globally.

Reply:
Interesting - wasn't aware of those recommendations. But yeah, agree consistency is probably better otherwise the empty files existence may be confusing.